### ISSUE TITLE
idea: eights part of idea violations

### ISSUE THREAD

**Issue by Nimfadora** [checkstyle/checkstyle#4711]
sub-task of #4681

too widespread to be included to the report:
Chained method calls **1679**
Constant on right side of comparison **1880**
Instance field access not qualified with this **3512**
Instance method call not qualified with this **11547**
Local variable or parameter can be final **3834**
Nested method call **6207**
Unqualified inner class access **754**
Unqualified static access **4421**

[report on all other violations](https://nimfadora.github.io/idea/suppressed_code_style.html)

violations names included:
```
ChainedMethodCall
ConstantOnRHSOfComparison
ImplicitCallToSuper
LocalCanBeFinal
NestedMethodCall
UnqualifiedFieldAccess
UnqualifiedInnerClassAccess
UnqualifiedMethodAccess
UnqualifiedStaticUsage
ReturnThis
```

**Comment by romani** [comment]
>Implicit call to 'super()'

Suppress : "we do not like this style, but we could change our mind in future"

>Return of 'this' inspection

All cases are false positives. If there are options in inspections to avoid them, lets suppress "all cases are false positives"

**Comment by romani** [comment]
> Violations included:

please activate that list of inspections and send PR(not for merge) to generate report on TC.
After each merge to our master, update that PR with list of item that still unresolved, TC will keep report for you.

**Comment by Nimfadora** [comment]
@romani this task is also done, these violations were not even included in PR with all remained violations.

**Comment by romani** [comment]
ok, am closing this issue as you confirming that all are covered. 
We will recheck what is not covered at end of unmbrella issue (#4681).
